pol-core/pol/network/pktboth.h

```diff
@@ -150,9 +151,9 @@ struct PKTBI_3B
     u8 layer;
     u32 item_serial;
     u16 number_bought;
-  } items[1]; /* Dunno how many it should be */
+  } items[( MAXBUFFER - 8 ) / 7];
 };
-static_assert( sizeof( PKTBI_3B ) == 15, "size missmatch" );
+static_assert( sizeof( PKTBI_3B ) == 2556, "size missmatch" );
```

Comment:
is there a better constant here than this magic number?

Reply:
And is this related to the 2559 in other spots?

Reply:
No just random :)